### PR TITLE
upgrade tx version from 1 to 2

### PR DESCRIPTION
**Pull request by ouwenkg**
# Description 

Two main issues solved in this protocol version upgrade.

1. Fix autoExec coinbase problem.
2. Fix suicide refund problem

### Applicable Issues

#508 
#477

### CODE REVIEW

**Comment by kaikai1024** [comment]
https://github.com/cryptape/cita-common/pull/270#issuecomment-491143063

File: scripts/txtool/txtool/make_tx.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def _blake2b_ed25519_deploy_data(current_height,
     if receiver is not None:
         if version == 0:
             tx.to = receiver
-        elif version == 1:
+        elif version == 1 or 2:
```
Comment (yangby-cryptape) [inline scripts/txtool/txtool/make_tx.py]:
```python
a = 3
if a == 1 or 2:
    print("hello")
```

Comment (ouwenkg) [reply scripts/txtool/txtool/make_tx.py]: O, my fault.

**Comment by kaikai1024** [comment]
> Fix suicide refund problem

An independent PR.

Then upgrade protocol version.(not tx version)

**Pull request closed without merging**